Makefile

```diff
@@ -58,15 +58,23 @@ docker-build-run:
 
 .PHONY: docker-run-deploy
 docker-run-deploy:
-	[ -f jenkins-env ] && cat jenkins-env | grep GIT > inherit-env
+	[ -f jenkins-env ] && cat jenkins-env | grep -e GIT -e DEVSHIFT > inherit-env
 	[ -f inherit-env ] && . inherit-env
+	REGISTRY="push.registry.devshift.net"
```

Comment:
educate me here: what is the diff between `push.registry.devshift.net` and `registry.devshift.net` ?